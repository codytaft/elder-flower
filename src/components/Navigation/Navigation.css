.nav-link-bar {
  display: flex;
  flex-direction: row;
  justify-content: center;
  padding: 0;
  width: 100%; }

.top-nav-link {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  width: 15rem;
  margin: 0 1.5rem 0 1.5rem;
  text-decoration: none;
  font-size: 32px;
  color: #4e2d1d;
  border-radius: 5rem;
  border: 0.1rem solid #4e2d1d; }

.top-nav-link:hover {
  background-color: #d7b88d; }

.nav-link {
  display: flex;
  flex-direction: row; }
