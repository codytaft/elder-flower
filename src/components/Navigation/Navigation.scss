.nav-link-bar {
  display: flex;
  flex-direction: row;
  justify-content: center;
  padding: 0;
  width: 100%;
}

.top-nav-link {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  width: 15rem;
  margin: 0 1.5rem 0 1.5rem;
  text-decoration: none;
  font-size: 32px;
  color: #4e2d1d;
  border-radius: 5rem;
  border: 0.1rem solid rgb(78, 45, 29);
}

.top-nav-link:hover {
  background-color: rgba(215, 184, 141, 1);
}

.nav-link {
  display: flex;
  flex-direction: row;
}
