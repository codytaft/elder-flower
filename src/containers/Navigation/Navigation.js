import React from 'react';
import { NavLink } from 'react-router-dom';

export const Navigation = {
  render() {
    return (
      <section className="nav-link-section">
        <NavLink exact to="/login" className="nav-link nav-link-login">
          Login
        </NavLink>
        <NavLink exact to="/sign-up-home" className="nav-link nav-link-signup">
          SignUp
        </NavLink>
      </section>
    );
  }
};

export default Navigation;
