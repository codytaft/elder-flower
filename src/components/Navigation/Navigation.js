import React from 'react';
import { NavLink } from 'react-router-dom';

import './Navigation.css';

export const Navigation = () => {
  return (
    <header className="nav-link-bar">
      <NavLink exact to="/contacts" className="top-nav-link">
        Contacts
      </NavLink>
    </header>
  );
};

export default Navigation;
