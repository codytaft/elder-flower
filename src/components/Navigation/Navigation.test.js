import React from 'react';
import { shallow } from 'enzyme';
import { Navigation } from './Navigation';

describe('Navigation', () => {
  it('should match the snapshot', () => {
    let wrapper;
    wrapper = shallow(<Navigation />);
    expect(wrapper).toMatchSnapshot();
  });
});
