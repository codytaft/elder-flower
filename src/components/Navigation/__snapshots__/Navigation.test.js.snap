// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        aria-current="page"
        className="side-nav-link nav-link-contacts"
        exact={true}
        to="/contacts"
>
        Contacts
</NavLink>,
      "className": "nav-link-bar",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "aria-current": "page",
        "children": "Contacts",
        "className": "side-nav-link nav-link-contacts",
        "exact": true,
        "to": "/contacts",
      },
      "ref": null,
      "rendered": "Contacts",
      "type": [Function],
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          aria-current="page"
          className="side-nav-link nav-link-contacts"
          exact={true}
          to="/contacts"
>
          Contacts
</NavLink>,
        "className": "nav-link-bar",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "Contacts",
          "className": "side-nav-link nav-link-contacts",
          "exact": true,
          "to": "/contacts",
        },
        "ref": null,
        "rendered": "Contacts",
        "type": [Function],
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
